robot/Cumulus/tests/browser/gift_entry/template_builder.robot

```diff
@@ -40,11 +44,13 @@ Create Clone and Delete Template
     Click Gift Entry Button                   Save & Close
     Wait Until Page Contains                  This name has been used by another template. Please enter a unique name.
     Enter Value In Field                      Template Name=${new_template}
+    Click Gift Entry Button                   Next: Form Fields
+    Click Gift Entry Button                   Next: Batch Settings
     Click Gift Entry Button                   Save & Close
     Current Page Should Be                    Landing                        GE_Gift_Entry
     Click Link                                Templates
     Wait Until Page Contains                  ${new_template}
-    
+    Store Template Record Id                  ${new_template}
```

Comment:
This step was not originally added as the template is being deleted in the next step but makes sense if the test fails before going to the next step then it gets deleted as part of teardown

Reply:
Yep, that was the intention with this. I'm seeing lots of cleanup failures when I run tests, so I'm working to ensure it gets deleted no matter what.